app/apps/workflow/models.py

```diff
@@ -462,7 +474,12 @@ def __str__(self):
         return f"{self.id}, case: {self.case.id}"
 
 
+USER_TASKS = {"task_create_debrief": {"due_date": datetime.timedelta(days=0)}}
+DEFAULT_USER_TASK_DUE_DATE = datetime.timedelta(days=2)
```

Comment:
Why 2 days? Is this a business rule?

Reply:
Arbitrary choice, a default should be used.